src/tests/test_continue.py

```diff
@@ -12,62 +12,97 @@
 from collections import OrderedDict
 from numpy import array
 
-class TestWaterTutorial(ForceBalanceTestCase):
-    def setUp(self):
-        super(ForceBalanceTestCase,self).setUp()
-        os.system('rm -rf test/files/test_continue.run')
-        os.makedirs('test/files/test_continue.run')
-        os.system('cp -r studies/001_water_tutorial/forcefield test/files/test_continue.run/forcefield')
-        os.system('cp -r studies/001_water_tutorial/targets test/files/test_continue.run/targets')
-        os.chdir('test/files')
-        os.system('cp -r test_continue.sav test_continue.in test_continue.tmp test_continue.run')
-        os.chdir('test_continue.run')
+# class TestWaterTutorial(ForceBalanceTestCase):
+class TestWaterTutirual:
+    # def setUp(self):
+    @classmethod
+    def setup_class(cls):
```

Comment:
Do we want to use `setup_class`, or a different setup that will run for each test (rather than all the tests in the same class)?

Reply:
I think it's best to use `setup_method`, so that setup and teardown happens _between each test_ rather than _before or after all of the tests_

Reply:
There are cases where it makes sense to have setup and teardown such that the workflow goes as setup-test1-test2-teardown, as opposed to setup1-test1-teardown1-setup2-test2-teardown2.

For example, if we set up a system in GROMACS and TINKER, and see if a number of properties match (e.g. gradients, optimized geometry, and vibrational frequency).  Because each property involves running different parts of the codes, it makes sense to have them as different tests.  However, it takes a small amount of time to set up the system so it makes sense to setup and teardown only once.

A related issue is that for the tests above, they should not have separate setup and teardown codes because that would introduce redundancy.  This may be resolvable separately from the computational cost concern.